libraries/radwordsprocessing/formats-and-conversion/word-file-formats/docx/docxformatprovider.md

```diff
@@ -1,18 +1,22 @@
 ---
 title: Using DocxFormatProvider
 page_title: Using DocxFormatProvider
-description: Using DocxFormatProvider
+description: Import and Export to Docx File Format Using DocxFormatProvider
```

Comment:
Learn how you can import and export documents from/to DOCX using WordsProcessing.